test/integration/scenarios/common.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const utils = require('../utils');
+
+function setMonitoringSocketsConnections(params, configurations) {
```

Comment:
can we move this function `setMonitoringSocketsConnections` to root common file? this way when we can avoid the duplication and confusion while requiring
```
const common = require('../common');
const networkCommon = require('./common');
```